src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -269,7 +269,7 @@ export class ViolationTicketService {
         ticketNumber: this.ticket.ticket_number,
         time: this.datePipe.transform(this.ticket.issued_date, "HH:mm")
       };
-      if (this.dateDiff(this.ticket.issued_date) <= 30) {
+      if ((this.dateDiff(this.ticket.issued_date) <= 30 && this.ticketType == 'E') || (this.dateDiff(this.ticket.issued_date) <= 45 && this.ticketType == 'S')) {
```

Comment:
ticketType is returning ticketTypes in "@shared/enums/ticket-type.enum"
the enum should be something like ticketTypes.ELECTRONIC_TICKET